<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gomoku</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,75,0,0">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="GameView" name="gameView">
        <property name="minimumSize">
         <size>
          <width>760</width>
          <height>760</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>760</width>
          <height>760</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="doubleThreeCheckBox">
          <property name="text">
           <string>Double Three</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="breakableFiveCheckBox">
          <property name="text">
           <string>Breakable Five</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Black:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>White:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="_blackScore">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="_whiteScore">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="logText">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="restartButton">
        <property name="text">
         <string>Restart (r)</string>
        </property>
        <property name="shortcut">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="undoButton">
        <property name="text">
         <string>Undo (u)</string>
        </property>
        <property name="shortcut">
         <string>U</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionStyle"/>
    <addaction name="actionDark_Style"/>
   </widget>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>Gomoku</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuQuit"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionStyle">
   <property name="text">
    <string>Custom Style</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDark_Style">
   <property name="text">
    <string>Dark Style</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GameView</class>
   <extends>QGraphicsView</extends>
   <header>gameview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
